src/lib/commands.js

```diff
@@ -88,5 +88,9 @@ exports.getCategories = () => {
       description: 'Other useful commands',
       commands: [],
     },
+    categories: {
```

Comment:
Can you move this above `utilities` so it these category commands won't be at the very end of the `--help` screen?